README.md

```diff
@@ -64,6 +64,15 @@ parameters:
         - '%kernel.project_dir%/vendor/project-without-composer'
 ```
 
+You can also exclude files or directories - use regex or [fnmatch](http://php.net/manual/en/function.fnmatch.php):
```

Comment:
We can simplify this link:
```diff
-http://php.net/manual/en/function.fnmatch.php
+https://php.net/fnmatch
```

Reply:
Might be, but that extra manual work.
I simply copy what's in the URL on the site.